Solutions/Cyware/Hunting queries/UnusualNetworkConnectionsToRareExternalDomains.yaml

```diff
@@ -0,0 +1,22 @@
+id: 61c99147-b749-4164-80b1-c4bfa4efa704
+name: Detecting Suspicious PowerShell Command Executions
+description: |
+  'Spot connections to rarely accessed external domains that are present in your watchlist, which could signify data exfiltration attempts or C2 communication.'
+requiredDataConnectors: [
+'DeviceProcessEvents'
+]
+tactics:
+  - Command And Control
```

Comment:
Hi @AashiqRamachandran, tactics names MUST NOT have any spaces. It should be `CommandAndControl`. Also relevantTechniques missing for this tactic.